src/nnet3/nnet-simple-component.h

```diff
@@ -309,6 +310,7 @@ class AffineComponent: public UpdatableComponent {
   Component *CollapseWithNext(const FixedAffineComponent &next) const;
```

Comment:
And please fix the Convolutional1dComponent even if it's deprecated.. we
don't want tests to fail.

On Tue, Nov 3, 2015 at 3:34 PM, Daniel Povey <dpovey@gmail.com> wrote:

> It writes to a VectorBase, which is not resizable... there are situations
> where it's writing to a sub-vector of a larger vector, which is why we
> designe it that way.
> Dan
>
>
> On Tue, Nov 3, 2015 at 3:32 PM, hhadian <notifications@github.com> wrote:
>
>> In src/nnet3/nnet-simple-component.h
>> <https://github.com/kaldi-asr/kaldi/pull/307#discussion_r43802846>:
>>
>> > @@ -309,6 +310,7 @@ class AffineComponent: public UpdatableComponent {
>> >    Component *CollapseWithNext(const FixedAffineComponent &next) const;
>>
>> Currently the Vectorize functions do not automatically resize the out
>> vector (*params). Isn't it better to automatically resize it to
>> NumParameters() ?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kaldi-asr/kaldi/pull/307/files#r43802846>.
>>
>
>